wiki/BanchoBot/ja.md

```diff
@@ -0,0 +1,47 @@
+<!-- BanchoBot 情報 -->
+2
```

Comment:
what the hell?
why the line number here?

Reply:
I'm sorry.
I fixed now.
(maybe it is my mistake)